client/package.json

```diff
@@ -1,28 +1,45 @@
 {
-  "name": "react-webpack-rails-tutorial",
-  "version": "0.0.1",
+  "name": "gov-au-transitions-beta",
+  "version": "1.0.0",
+  "private": true,
+  "description": "BETA version of transitions aged care app",
   "engines": {
     "node": "5.10.0",
     "npm": "3.5.0"
   },
   "scripts": {
     "build:test": "webpack --config webpack.config.js",
     "build:production": "NODE_ENV=production webpack --config webpack.config.js",
-    "build:development": "webpack -w --config webpack.config.js"
+    "build:development": "webpack -w --config webpack.config.js",
+    "test": "ava"
   },
-  "cacheDirectories": ["node_modules", "client/node_modules"],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/AusDTO/govau-transitions-beta.git"
+  },
+  "author": "transitions team",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/AusDTO/govau-transitions-beta/issues"
+  },
+  "homepage": "https://github.com/AusDTO/govau-transitions-beta#readme",
+  "cacheDirectories": [
+    "node_modules",
+    "client/node_modules"
+  ],
   "dependencies": {
     "babel": "^6.5.2",
     "babel-cli": "^6.6.5",
     "babel-core": "^6.7.4",
     "babel-loader": "^6.2.4",
-    "babel-runtime": "^6.6.1",
     "babel-polyfill": "^6.7.4",
     "babel-preset-es2015": "^6.6.0",
     "babel-preset-react": "^6.5.0",
     "babel-preset-stage-0": "^6.5.0",
+    "babel-runtime": "^6.11.6",
     "es5-shim": "^4.5.7",
     "expose-loader": "^0.7.1",
+    "gov-au-ui-kit": "git+https://github.com/nathf/gov-au-ui-kit.git#feature/node-install-issues",
```

Comment:
This ref is only temporary until some node install issues are resolved on the ui-kit side